src/init.js

```diff
@@ -178,7 +178,7 @@ export const init = async (options) => {
 		const { siteId, secretKey } = await auth.saveSecretKey(answers.secretKey, answers.siteId);
 
 		await setRepoSecret(options, { siteId: answers.siteId, secretKey: answers.secretKey, organization: answers.organization, name: answers.name });
-		console.log();
```

Comment:
This was here for spacing - maybe return it or add some spacing another way?

See spacing missing now before "The testing-branch-protection directory...":
![image](https://user-images.githubusercontent.com/5060400/148465759-30764aeb-2dfb-432e-a91d-13d9734fc880.png)